pallets/loans/src/mock.rs

```diff
@@ -281,14 +281,21 @@ pub(crate) fn run_to_block(n: BlockNumber) {
     Loans::on_finalize(System::block_number());
     for b in (System::block_number() + 1)..=n {
         System::set_block_number(b);
-        Timestamps::set_timestamp(6000 * b);
         Loans::on_initialize(System::block_number());
```

Comment:
looks like we don't have `on_initialize` anymore right?

Reply:
Yes. I leave the function here to maintain a test framework.